lua/nvim-tree/actions/dispatch.lua

```diff
@@ -25,6 +25,7 @@ local Actions = {
   trash = require("nvim-tree.actions.fs.trash").fn,
   remove = require("nvim-tree.actions.fs.remove-file").fn,
   rename = require("nvim-tree.actions.fs.rename-file").fn(false),
+  relative_rename = require("nvim-tree.actions.fs.rename-file").fn(false, true),
```

Comment:
Please add a default action to `lua/nvim-tree/actions/init.lua` and run `scripts/update-help.sh`

`<c-r>` is taken; what do you think about `"e"` ?

That will address the documentation. "in-vim" documentation is coming #1579

Reply:
Thanks - and yes, I'm good with e - it's good to have the different renames available at different key strokes.

I had to change `gsed` instead of `sed` in the `update-help.sh` for it to run (I'm currently on a mac, and mac sed behaves different to gnu sed), but I didn't commit that script change.